app/controllers/api/v0/classified_listings_controller.rb

```diff
@@ -16,10 +16,13 @@ class ClassifiedListingsController < ApiController
       def index
         @classified_listings = ClassifiedListing.published.
           select(ATTRIBUTES_FOR_SERIALIZATION).
-          includes(:user, :organization, :taggings)
-
-        @classified_listings = @classified_listings.where(category: params[:category]) if params[:category].present?
+          includes(:user, :organization, :taggings, :classified_listing_category)
 
+        if params[:category].present?
+          category = ClassifiedListingCategory.find_by(slug: params[:category])
```

Comment:
if the category is not found, it will return all listings, is that intended?

Reply:
That's the same behavior as before:

```ruby
@classified_listings = @classified_listings.where(category: params[:category]) if params[:category].present?
```

 `params[:category]` will only be present of we filter.